lmdeploy/model.py

```diff
@@ -1324,6 +1324,45 @@ def match(cls, model_path: str) -> Optional[str]:
             return 'internvl-zh-hermes2'
 
 
+@MODELS.register_module(name='phi-3')
+class Phi3Instruct(BaseChatTemplate):
+    """Chat template of InternLM model."""
+
+    def __init__(
+            self,
+            system='<|system|>\n',
+            meta_instruction="""You are a helpful AI assistant.""",  # noqa: E501
```

Comment:
where is this meta_instruction from?

Reply:
https://huggingface.co/microsoft/Phi-3-mini-128k-instruct/commit/fc313b0514906a348a624cc9421e36f52c60e137